src/pages/policies/[policyId]/claims/[claimId].svelte

```diff
@@ -77,11 +78,11 @@ $: claimItem = claim.claim_items?.[0] || ({} as ClaimItem) //For now there will
 $: setInitialValues(claimItem)
 $: statusText = getClaimStatusText(claim, claimItem)
 
-$: items = $itemsByPolicyId[policyId] || []
+$: items = $selectedPolicyItems
 $: policyId && loadItems(policyId)
 $: item = items.find((itm) => itm.id === claimItem.item_id) || ({} as PolicyItem)
 
-$: isMemberOfPolicy = itemBelongsToPolicy($user.policy_id, item)
+$: isMemberOfPolicy = memberBelongsToPolicy($user.id, $policies, item.policy_id)
```

Comment:
It looks like this page can get its Policy ID from three different sources:
1. from routify injecting it (`export let policyId...`),
2. from our store (`... let policyId = $selectedPolicyId`), or
3. from an item (`memberBelongsToPolicy(..., ..., item.policy_id)`, derived from the claimId in the URL).

If we can reduce the number of places we look for a Policy ID, I think it will reduce odd bugs and ease troubleshooting.
```suggestion
$: isMemberOfPolicy = memberBelongsToPolicy($user.id, $policies, policyId)
```

Thoughts?

Reply:
fixed. i was trying to think if there was any case when the api might return us policy items where the policy_id is not what we just retrieved from. probably not worthwhile worrying about. :)